src/cli_verification_code/cli_verification_code.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { kibanaPackageJson, getConfigDirectory } from '@kbn/utils';
+import path from 'path';
+import fs from 'fs';
+import chalk from 'chalk';
+
+import Command from '../cli/command';
+
+const program = new Command('bin/kibana-verification-code');
+
+program
+  .version(kibanaPackageJson.version)
+  .description('Tool to get Kibana verification code')
+  .action(() => {
+    const fpath = path.join(getConfigDirectory(), '.code');
```

Comment:
__question:__ shouldn't we use `getConfigDirectory` when we write the file as well?